### PR TITLE
Update documentation

### PR DESCRIPTION
**Pull request by imobachgs**
Update README.md and add a DEVELOPMENT.md document with some tips/hints about the module's development.

## TODO

- [x] Update the screenshot

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -13,32 +13,55 @@ of basic uses cases:
...
+[watch mode](https://webpack.js.org/guides/development/#using-watch-mode) to
+automatically update the webpack on every code change with
+
+```
+npm run watch
+```
+or
+```
+make watch
+```
```
Comment (dgdavid) [inline README.md:58]: Those lines can be moved to the `Developing` section, can't they?

**Review by dgdavid — APPROVED** [review]
Looks good, just a minor suggestion

File: README.md (hunk 1 above)
Comment (imobachgs) [reply README.md:58]: Good point. But I would prefer to point from this document to DEVELOPMENT.md.

**Review by lslezak — APPROVED** [review]
Nice! :+1:

**Pull request merged**